### ISSUE TITLE
The provided execution role does not have permissions to call CreateNetworkInterface on EC2

### ISSUE THREAD

**Issue by ian-lmry** [webiny/webiny-js#2580]
### Version

5.30.0

### What are the steps to reproduce this bug?

Deploy a webiny project created via `npx create-webiny-project <folder>` to AWS using `yarn webiny deploy --env prod`

### What is the expected behavior?

The DEV deployment using the same project was successful. However this issue persists on PROD after many retries. Have also tried using a fresh environment and the same problem persists.

### What do you see instead?

```
Diagnostics:
  aws:lambda:Function (ps-render-lambda):
    error: 1 error occurred:
        * error creating Lambda Function (1): InvalidParameterValueException: The provided execution role does not have permissions to call CreateNetworkInterface on EC2
    {
      RespMetadata: {
        StatusCode: 400,
        RequestID: "5502525f-0a57-41c9-9f9f-d4181ac990fd"
      },
      Message_: "The provided execution role does not have permissions to call CreateNetworkInterface on EC2",
      Type: "User"
    }
 
  aws:lambda:Function (ps-render-subscriber-lambda):
    error: 1 error occurred:
        * error creating Lambda Function (1): InvalidParameterValueException: The provided execution role does not have permissions to call CreateNetworkInterface on EC2
    {
      RespMetadata: {
        StatusCode: 400,
        RequestID: "0cbc21a1-840b-452a-a107-2af64e3f03ff"
      },
      Message_: "The provided execution role does not have permissions to call CreateNetworkInterface on EC2",
      Type: "User"
    }
 
  aws:lambda:Function (ps-flush-lambda):
    error: 1 error occurred:
        * error creating Lambda Function (1): InvalidParameterValueException: The provided execution role does not have permissions to call CreateNetworkInterface on EC2
    {
      RespMetadata: {
        StatusCode: 400,
        RequestID: "15e71ac2-4a51-4422-b565-f2b27e7756e9"
      },
      Message_: "The provided execution role does not have permissions to call CreateNetworkInterface on EC2",
      Type: "User"
    }
 
  pulumi:pulumi:Stack (website-prod):
    error: update failed
```

**Comment by Pavel910** [comment]
Interesting, thanks, we'll test this.

**Comment by ian-lmry** [comment]
> 

Thank you, let me know if you need any further information. In the meantime I'm playing around with adding the correct policy although I've never used pulumi before so it's a Friday adventure!

https://stackoverflow.com/questions/41177965/aws-lambdathe-provided-execution-role-does-not-have-permissions-to-call-describ/54668725#54668725

**Comment by Pavel910** [comment]
@ian-lmry I looked into this, and it will be fixed in 5.31.0 release coming up this week. Many thanks for reporting this issue! 🙌 
In short, the issue is with execution roles we assign to those Lambda functions when VPC is enabled (and VPC is enabled in a `prod` environment).
